.rubocop.yml

```diff
@@ -0,0 +1,21 @@
+Metrics/LineLength:
+  Enabled: False
+
+# Offense count: 1
+# Configuration parameters: CountComments.
+Metrics/MethodLength:
+  Max: 13
+
+# Offense count: 12
+Style/AsciiComments:
+  Enabled: false
+
```

Comment:
Was also gonna add:

```
Style/SymbolArray:
  Enabled: true
```

Since we use ruby 2.3.1, we can just enforce symbol keys in hashes